bib/cmd/bootc-image-builder/main.go

```diff
@@ -640,11 +641,29 @@ func run() error {
 	// "quay.io" will create an "err != nil". Ideally we could check err
 	// for something like cobra.UnknownCommandError but cobra just gives
 	// us an error string
-	_, _, err := rootCmd.Find(os.Args[1:])
-	if err != nil && !slices.Contains([]string{"help", "completion"}, os.Args[1]) {
+	cmd, _, err := rootCmd.Find(os.Args[1:])
+	injectBuildArg := func() {
 		args := append([]string{buildCmd.Name()}, os.Args[1:]...)
 		rootCmd.SetArgs(args)
 	}
+	// command not known, i.e. happens for "bib quay.io/centos/..."
```

Comment:
This is slightly ugly but I couldn't find a better way than to handle the two cases explicitly

Reply:
:shrug: I have no better idea.